### PR TITLE
Add solutions backup monitor and example cryptpad

### PR DESCRIPTION
**Pull request by waleedhammam**
### Description

- Creating a production infrastructure with backups

### Related Issues

- https://github.com/threefoldtech/js-sdk/issues/1395

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings

### CODE REVIEW

File: jumpscale/tools/restic/restic.py [hunk 1]
```diff
@@ -91,8 +91,32 @@
...
+
+latest_backup=`restic snapshots --last -c -q | sed -n 3p |cut -d " " -f3`
+latest_backup_epoch=`date "+%s" -d "$latest_backup"`
+wait $!
+
+current_date=`date +"%s"`
+
+if [[ $(( (current_date - latest_backup_epoch) / 86400 )) > 2 ]]
+then
+    echo "send mail"
```
Comment (ranatrk) [inline jumpscale/tools/restic/restic.py:111]: shouldn't we send an email here?

Comment (waleedhammam) [reply jumpscale/tools/restic/restic.py:111]: yes, we will add it, thanks 👍🏻

**Review by ranatrk — APPROVED** [review]

File: jumpscale/packages/tfgrid_solutions/chats/cryptpad_advanced.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+            enforce_https=False,
+            proxy_pool_id=self.gateway_pool.pool_id,
+            node_id=self.selected_node.node_id,
+            solution_uuid=self.solution_id,
+            log_config=self.nginx_log_config,
+            **self.solution_metadata,
+        )
+        success = deployer.wait_workload(_id, self)
+        if not success:
+            # solutions.cancel_solution(self.workload_ids)
```
Comment (dmahmouali) [inline jumpscale/packages/tfgrid_solutions/chats/cryptpad_advanced.py:140]: should this line uncomment?

Comment (waleedhammam) [reply jumpscale/packages/tfgrid_solutions/chats/cryptpad_advanced.py]: no, removed already

**Review by dmahmouali — APPROVED** [review]

**Review by xmonader — APPROVED** [review]

**Pull request merged**